Gems/ScriptCanvasPhysics/Code/Tests/ScriptCanvasPhysicsTest.cpp

```diff
@@ -175,7 +175,7 @@ namespace ScriptCanvasPhysicsTests
         MOCK_CONST_METHOD0(GetNativePointer, void*());
     };
 
-    class MockShape        
+    class MockShape
```

Comment:
This class may need
```
virtual ~MockShape() = default;
```

Reply:
it doesnt because the base (`Physics::Material`) has a virtual destructor